doc/source/migrate_from_ppa.md

```diff
@@ -0,0 +1,89 @@
+:orphan:
```

Comment:
Can we link to this instead of using `:orphan:`?

Reply:
In the following PR, I'm thinking I will add a link to this page in `ubuntu/install.md`.
Or should we add this page under the other page?

Reply:
OK.

BTW, why do we want to create a new page instead of adding this to `ubuntu/install.md`? Do we want to reuse this page from multiple pages?

Reply:
I originally thought that installation and repository migration might be separate topics. 
But since the repository migration is related to installation, there's no need to separate them.
I'll add a section at the end of `ubuntu/install.md` with this content.

Reply:
fix: [2f67ddc](https://github.com/groonga/groonga/pull/2260/commits/2f67ddc83d35f7ff6dbb7cbb69358204ae526fc0) I've moved the migration contents to `ubuntu/install.md`